tests/beta/test_default_tensor.py

```diff
@@ -795,189 +972,309 @@ def circuit():
+    def test_supported_state_preparation(self, rep, tol, name, par, expected_output):
         """Tests supported state preparations"""
 
         op = getattr(qml.ops, name)
 
-        assert tensornet_device_2_wires.supports_operation(name)
+        dev = qml.device("default.tensor", wires=2, representation=rep)
 
-        @qml.qnode(tensornet_device_2_wires)
+        assert dev.supports_operation(name)
+
+        @qml.qnode(dev)
         def circuit():
             op(np.array(par), wires=[0, 1])
             return qml.expval(qml.PauliZ(0)), qml.expval(qml.PauliZ(1))
 
         assert np.allclose(circuit(), expected_output, atol=tol, rtol=0)
 
     # This test is ran on the state |0> with one Z expvals
-    @pytest.mark.parametrize("name,par,expected_output", [
-        ("PhaseShift", [math.pi/2], 1),
-        ("PhaseShift", [-math.pi/4], 1),
-        ("RX", [math.pi/2], 0),
-        ("RX", [-math.pi/4], 1/math.sqrt(2)),
-        ("RY", [math.pi/2], 0),
-        ("RY", [-math.pi/4], 1/math.sqrt(2)),
-        ("RZ", [math.pi/2], 1),
-        ("RZ", [-math.pi/4], 1),
-        ("Rot", [math.pi/2, 0, 0], 1),
-        ("Rot", [0, math.pi/2, 0], 0),
-        ("Rot", [0, 0, math.pi/2], 1),
-        ("Rot", [math.pi/2, -math.pi/4, -math.pi/4], 1/math.sqrt(2)),
-        ("Rot", [-math.pi/4, math.pi/2, math.pi/4], 0),
-        ("Rot", [-math.pi/4, math.pi/4, math.pi/2], 1/math.sqrt(2)),
-        ("QubitUnitary", [np.array([[1j/math.sqrt(2), 1j/math.sqrt(2)], [1j/math.sqrt(2), -1j/math.sqrt(2)]])], 0),
-        ("QubitUnitary", [np.array([[-1j/math.sqrt(2), 1j/math.sqrt(2)], [1j/math.sqrt(2), 1j/math.sqrt(2)]])], 0),
-    ])
-    def test_supported_gate_single_wire_with_parameters(self, tensornet_device_1_wire, tol, name, par, expected_output):
+    @pytest.mark.parametrize(
+        "name,par,expected_output",
+        [
+            ("PhaseShift", [math.pi / 2], 1),
+            ("PhaseShift", [-math.pi / 4], 1),
+            ("RX", [math.pi / 2], 0),
+            ("RX", [-math.pi / 4], 1 / math.sqrt(2)),
+            ("RY", [math.pi / 2], 0),
+            ("RY", [-math.pi / 4], 1 / math.sqrt(2)),
+            ("RZ", [math.pi / 2], 1),
+            ("RZ", [-math.pi / 4], 1),
+            ("Rot", [math.pi / 2, 0, 0], 1),
+            ("Rot", [0, math.pi / 2, 0], 0),
+            ("Rot", [0, 0, math.pi / 2], 1),
+            ("Rot", [math.pi / 2, -math.pi / 4, -math.pi / 4], 1 / math.sqrt(2)),
+            ("Rot", [-math.pi / 4, math.pi / 2, math.pi / 4], 0),
+            ("Rot", [-math.pi / 4, math.pi / 4, math.pi / 2], 1 / math.sqrt(2)),
+            (
+                "QubitUnitary",
+                [
+                    np.array(
+                        [
+                            [1j / math.sqrt(2), 1j / math.sqrt(2)],
+                            [1j / math.sqrt(2), -1j / math.sqrt(2)],
+                        ]
+                    )
+                ],
+                0,
+            ),
+            (
+                "QubitUnitary",
```

Comment:
lol `black` we love you

Reply:
🤯

Reply:
Nitpick: I tend to prefer having large lists of test data as module or class variables. Even though it's a bit further away from the tests, having to scroll for test data is much less overhead than having to scroll for logic, and it makes the test file easier to read/avoids additional indentation.

Reply:
hmm... I don't know if I see any advantages to moving these further away from the tests. Can you elaborate more?